lib/install/management/delete.go

```diff
@@ -321,7 +321,7 @@ func (d *Dispatcher) DeleteVCHInstances(conf *config.VirtualContainerHostConfigS
 				errs = append(errs, err.Error())
 				mu.Unlock()
 			}
-			d.op.Debugf("Successfully deleted container: %s", child.InventoryPath)
+			d.op.Debugf("Successfully deleted container: %s", child.Reference().String())
```

Comment:
This has changed because InventoryPath is often empty. I think `ObjectName` is going to be more reliable, thanks!

Reply:
Is it worth going back to the property collector to populate the name field of this log message? child.ObjectName will have to go back to vsphere to get the name

Reply:
@zjs thoughts on the above?

Reply:
If we deleted it, we won't be able to go back and get the name will we?

Reply:
hmmm, yeah we should not actually be able to do that. We could get it before hand. I was just curious of the merit of this log line. I am  all for it, but to get the name we are going to have to query vsphere again. I was trying to decide if that was worth it.

Reply:
We also log the Reference in most of the other places. which is why I opted for the reference here. Though I agree with @hickeng the reference is almost useless.

Reply:
It looks like `d.deleteVM` already calls `vm.ObjectName` in a few places, mostly for error handling.

I wonder if it'd be easier just to call `vm.ObjectName` once at the beginning of `d.deleteVM`, update the error logging to use that already-retrieved name, and move this success log into `d.deleteVM` where it could use that same name.

(In general, I think there's value in explicitly logging what we've deleted. For example, it makes it easier for someone wondering whether a missing VM was deleted by VIC to verify whether or not that's what happened to it.)